Chirp/Login Page/LoginPageViewController.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  LoginPageViewController.swift
+//  Chirp
+//
+//  Created by ioannis on 7/6/23.
+//
+
+import UIKit
+import PhoneNumberKit
+
+class LoginPageViewController: UIViewController {
+  
+    private let titleLabel:UILabel = {
+        let label = UILabel()
+        label.text = "What’s your phone number?"
+        label.numberOfLines = 0
+        label.font = .systemFont(ofSize: 32,weight:.bold)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .label
+        return label
+    }()
+    
+    private let subTitleLabel:UILabel = {
+        let label = UILabel()
+        label.text = "We’ll send you a code to verify your number"
+        label.numberOfLines = 0
+        label.font = .systemFont(ofSize: 14,weight:.light)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .label
+        return label
+    }()
+    private let numberTextField : PhoneNumberTextField = {
+       let textField = PhoneNumberTextField()
+        let phoneNumberKit = PhoneNumberKit()
+        textField.borderStyle = .roundedRect
+        textField.keyboardType = .namePhonePad
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        textField.placeholder = "Enter your mobile number"
+        textField.withFlag = true
+       
+        return textField
+    }()
+    
+    private let proceedButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setTitle("Proceed", for: .normal)
+        button.tintColor = .white
+        button.titleLabel?.font = .systemFont(ofSize: 16, weight: .bold)
+        button.backgroundColor =  .brown
+       // button.backgroundColor =  UIColor(red: 86, green: 49, blue: 37, alpha: 1)
+        button.layer.masksToBounds = true
+        button.layer.cornerRadius = 10
+        button.addTarget(self, action:  #selector(loginPressed), for: .touchUpInside)
+        return button
+    }()
```

Comment:
This will no longer be necessary. We can now use the custom button that @uzoh created.
```suggestion
    private let proceedButton: Button = {
        let button = Button()
        button.translatesAutoresizingMaskIntoConstraints = false
        button.setTitle("Proceed", for: .normal)
        button.addTarget(self, action:  #selector(loginPressed), for: .touchUpInside)
        return button
    }()
```